src/datomic_rest_api/rest/fields/gene.clj

```diff
@@ -2085,81 +2087,119 @@
      :description
      "gene models for this gene"}))
 
-
-
 ;;
 ;; Sequence Features widget
 ;;
 
 (defn associated-features [gene]
-  (let [db (d/entity-db gene)]
-    {:data
-     (->>
-       (q '[:find [?f ...]
-            :in $ ?gene
-            :where [?fg :feature.associated-with-gene/gene ?gene]
-                   [?f :feature/associated-with-gene ?fg]]
-          db (:db/id gene))
-       (map
-         (fn [fid]
-           (let [feature (entity db fid)]
-             (vmap
-               :name (pack-obj "feature" feature)
-               :description (first (:feature/description feature))
-               :method (-> (:feature/method feature)
-                           (:method/id))
-               :interaction (->> (:interaction.feature-interactor/_feature feature)
-                                 (map #(pack-obj "interaction" (:interaction/_feature-interactor %)))
-                                 (seq))
-               :expr_pattern (->>
-                               (q '[:find [?e ...]
-                                    :in $ ?f
-                                    :where [?ef :expr-pattern.associated-feature/feature ?f]
-                                           [?e :expr-pattern/associated-feature ?ef]
-                                           [?e :expr-pattern/anatomy-term _]]
-                                  db fid)
-                               (map
-                                 (fn [eid]
-                                   (let [expr (entity db eid)]
-                                     {:text (map #(pack-obj "anatomy-term" (:expr-pattern.anatomy-term/anatomy-term %))
-                                                 (:expr-pattern/anatomy-term expr))
-                                      :evidence {:by (pack-obj "expr-pattern" expr)}})))
-                               (seq))
-               :bound_by (->> (:feature/bound-by-product-of feature)
-                              (map #(pack-obj "gene" (:feature.bound-by-product-of/gene %)))
-                              (seq))
-               :tf  (pack-obj "transcription-factor" (:feature/transcription-factor feature))))))
-       (seq))
+  (let [db (d/entity-db gene)
```

Comment:
another minor one- perhaps use the required name everywhere when using datomic.api (`d/entity` vs  `entity`, `d/q` vs `q)